### PR TITLE
update order

### PR DESCRIPTION
**Pull request by meetar**
plus misc fixes + cleanup

### CODE REVIEW

File: shaders/position.yaml [hunk 1]
```diff
@@ -128,10 +129,11 @@ layers:
         draw:
             polygons:
                 style: buildings
-                order: 4
+                order: function() { return feature.sort_key; }
                 color: [.7,.7,.7]
                 extrude: true
             outline:
+                order: function() { return feature.sort_key + 1; }
```
Comment (bcamper) [inline shaders/position.yaml:138]: This is probably backwards -- if you draw an outline over the intended inline, you won't see the inline.

File: procedural/tronish.yaml [hunk 2]
```diff
@@ -115,24 +115,24 @@ layers:
...
         data:
             source: osm
         draw:
             tiledCross:
-                order: 2
+                order: function() { return feature.sort_key; }
                 color: [0,0,0]
             outline:
-                order: 3
+                order: function() { return feature.sort_key + 1; }
```
Comment (bcamper) [inline procedural/tronish.yaml:135]: -1 instead of +1?

File: procedural/tronish.yaml [hunk 3]
```diff
@@ -192,10 +191,10 @@ layers:
             source: osm
         draw:
             walls:
-                order: 6
+                order: function() { return feature.sort_key; }
                 color: [0, 0, 0]
             outline:
-                order: 7
+                order: function() { return feature.sort_key + 1; }
```
Comment (bcamper) [inline procedural/tronish.yaml:197]: -1 instead of +1?

File: shaders/color.yaml [hunk 4]
```diff
@@ -138,11 +138,11 @@ layers:
         draw:
             polygons:
                 style: buildings
-                order: 4
+                order: function() { return feature.sort_key; }
                 color: white
-                # color: Style.color.pseudoRandomColor
                 extrude: true
             outline:
+                order: function() { return feature.sort_key + 1; }
```
Comment (bcamper) [inline shaders/color.yaml:145]: -1 instead of +1?

File: shaders/position.yaml (hunk 1 above)
Comment (meetar) [inline shaders/position.yaml:138]: All of the examples here are outlines on top of polygons styles, so unless I'm misreading this there's no 'inline', just polygon fills. Or do polygon outlines work differently than I'm remembering?

Comment (bcamper) [inline shaders/position.yaml:138]: Oh sorry! Yes that makes sense :)  I'm stuck in line-outline-land. Sorry...

**Pull request merged**